src/ipc-handlers.ts

```diff
@@ -1084,3 +1085,11 @@ export function addSyncOperation( event: IpcMainInvokeEvent, id: string ) {
 export function clearSyncOperation( event: IpcMainInvokeEvent, id: string ) {
 	ACTIVE_SYNC_OPERATIONS.delete( id );
 }
+
+export function getSiteSize( _event: IpcMainInvokeEvent, siteId: string ) {
```

Comment:
Could we be specific about getting the size of wp-content in the function name? For example getWpContentSize, getWpContentSiteSize or getWpContentDirectorySize.

```suggestion
export function getWpContentSize( _event: IpcMainInvokeEvent, siteId: string ) {
```

Reply:
I improved the function naming in https://github.com/Automattic/studio/pull/811/commits/8babff9ba267cd5a5a74a233e53af339d062a9d4